contracts/src/_testing/unit/messaging/TestL2MessageManager.sol

```diff
@@ -32,8 +32,7 @@ contract TestL2MessageManager is Initializable, L2MessageManager, IGenericErrors
     __PauseManager_init(_pauseTypeRoles, _unpauseTypeRoles);
 
     _grantRole(DEFAULT_ADMIN_ROLE, msg.sender);
-    _grantRole(PAUSE_ALL_ROLE, _pauserManager);
-    _grantRole(UNPAUSE_ALL_ROLE, _pauserManager);
+    _grantRole(SECURITY_COUNCIL_ROLE, _securityCouncil);
```

Comment:
By removing the following, there is now no account account that can pause all or unpause all - please add it back.

```
_grantRole(PAUSE_ALL_ROLE, _pauserManager);
_grantRole(UNPAUSE_ALL_ROLE, _pauserManager);
```